twine-upload.sh

```diff
@@ -40,9 +40,9 @@ INPUT_VERIFY_METADATA="$(get-normalized-input 'verify-metadata')"
 INPUT_SKIP_EXISTING="$(get-normalized-input 'skip-existing')"
 INPUT_PRINT_HASH="$(get-normalized-input 'print-hash')"
 
-PASSWORD_DEPRECATION_NUDGE="::error title=Password-based uploads deprecated::\
-Starting in 2024, PyPI will require all users to enable Two-Factor \
-Authentication. This will consequently require all users to switch \
+PASSWORD_DEPRECATION_NUDGE="::error title=Password-based uploads disabled::\
+As of 2024, PyPI requires all users to enable Two-Factor \
+Authentication. This consequentlys require all users to switch \
```

Comment:
@woodruffw is this a typo?
```suggestion
Authentication. This consequently require all users to switch \
```

Reply:
Yes, but it's actually a double typo 😅 

```suggestion
Authentication. This consequently requires all users to switch \
```

Reply:
Oh, I didn't even notice the following one :)